extensions/security/deployment/pom.xml

```diff
@@ -35,6 +35,11 @@
             <artifactId>quarkus-junit5-internal</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-deployment</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Unfortunately this dependency introduces a cycle. Maybe you can suggest a different approach. It appears that CDI injection is not available to `QuarkusDevModeTest`-based tests.

Reply:
I think you should only use `quarkus-resteasy` in the test scope, have a look please for ex at `smallrye-jwt/deployment/pom.xml`

Reply:
I couldn't find the dependency you were referring to. But I don't think it will be possible to have this test in the security-deployment module so I instead moved the test to `integration-tests/devmode`.